newrelic/core/config.py

```diff
@@ -902,6 +902,12 @@ def default_otlp_host(host):
 _settings.application_logging.forwarding.context_data.enabled = _environ_as_bool(
     "NEW_RELIC_APPLICATION_LOGGING_FORWARDING_CONTEXT_DATA_ENABLED", default=False
 )
+_settings.application_logging.forwarding.context_data.include = _environ_as_bool(
+    "NEW_RELIC_APPLICATION_LOGGING_FORWARDING_CONTEXT_DATA_INCLUDE", default=""
```

Comment:
We don't have env vars for any of the other attribute include/excludes. I wonder if we should include env vars at all for these?

Reply:
We could, but I'm not inclined to make that change with this PR.

Reply:
I think you may have miss-understood me here. I was suggesting that we could delete these env vars and not provide them since we don't provide them for any of the other attribute settings but if you want to keep these that's fine too.